lib/framework.php

```diff
@@ -8,6 +8,7 @@
 
 define('VERSION', .1);
 
+require_once VENDOR_PATH.'composer/autoload_classmap.php';
```

Comment:
Individual modules require VENDOR_PATH/autoload.php when needed. If you need it here in order to use dotenv package, please require the autoload and not the autoload_classmap file.

Reply:
Also, I think you should remove the require blocks for vendor/autoload in modules.

Reply:
Copy that @kroky,

They have been removed in the latest commit.